src/bin/cargo-flamegraph.rs

```diff
@@ -210,25 +230,21 @@ fn build(opt: &Opt) {
                 continue;
             };
 
-            // Since workload() returns a Vec of paths, artifact.target.name could be contained in
-            // the path (e.g. in the project name). Thus .ends_with() is required to ensure the
-            // actual binary name is matched.
-            if workload(opt)
-                .iter()
-                .any(|w| w.ends_with(&artifact.target.name))
-                && artifact.profile.debuginfo.unwrap_or(0)
-                    != 0
+            // - Check that this is a binary we are interested in.
+            //   This should start with the target name (binaries can have names in format
+            //   `benchmark-deadbeef`)
+            // - The iterator is reversed because the entities we are interested in are most likely
+            //   to appear in the of the list.
```

Comment:
This sentence is missing something "are most likely to appear in the [?] of the list".

Reply:
In the beginning of the list. Thanks for noting that.